rts/lra/lra-filters/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -270,7 +270,7 @@ public void filter(ContainerRequestContext containerRequestContext) throws IOExc
 //                    previous = AtomicAction.suspend();
                 suspendedLRA = incommingLRA;
                 lraTrace(containerRequestContext, suspendedLRA, "ServerLRAFilter before: REQUIRES_NEW start new LRA");
-                newLRA = lraId = startLRA(incommingLRA, method, getTimeOut(method));
+                newLRA = lraId = startLRA(nested ? incommingLRA : null, method, getTimeOut(method));
```

Comment:
just thinking if it is better readable to use
```
if(nested) {
...
}
```
as it's used in other `case` branches in the code around. It just came to my mind when I'm looking at the PR. It could be subjective.

Reply:
I think that this would be easier to read:
```
String value = nested ? incommingLRA : null;
newLRA = lraId = startLRA(value, method, getTimeOut(method));
```
As there are different stylistic ways to do this and they have different merits I don't think we can force a particular approach here.

Reply:
Is this second coding style satisfactory?

Reply:
I think all three styles are valid so we should go with the authors choice

Reply:
no problem, I agree it's the authors choice

Reply:
If we think that certain java language constructs should not be used then we should update the style checker.